px-map-behavior-layer-geojson.es6.js

```diff
@@ -149,29 +198,46 @@
 
         style: (feature) => {
           const featureProperties = feature.properties.style || {};
+          const attributeProperties = this.getInstOptions().featureStyle;
 
-          return this._getStyle(featureProperties, styleAttributeProperties);
+          return this._getStyle(featureProperties, attributeProperties);
         }
       });
+
       if(this.editable) {
-        if (!this.parentNode.elementInst.editTools) {
-          this.parentNode.elementInst.editTools = new L.Editable(this.parentNode.elementInst, {featuresLayer: geojsonLayer});
-        } else {
-          this.parentNode.elementInst.editTools.featuresLayer.addLayer(geojsonLayer);
+        if(!this.parentNode.elementInst.editTools) {
```

Comment:
I would store this.parentNode.elementInst in a variable as it is used many times in this code.